src/m365/tenant/commands/serviceannouncement/serviceannouncement-message-get.ts

```diff
@@ -0,0 +1,76 @@
+import { Logger } from '../../../../cli';
+import GraphCommand from '../../../base/GraphCommand';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  id: string;
+}
+
+class TenantServiceAnnouncementMessageGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.SERVICEANNOUNCEMENT_MESSAGE_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieves a specified service update message for the tenant';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['startDateTime', 'endDateTime', 'lastModifiedDateTime', 'title', 'id', 'category', 'severity', 'tags', 'isMajorChange', 'actionRequiredByDateTime', 'services', 'expiryDateTime', 'hasAttachments', 'viewPoint' ];
+  }
+
+  public commandAction(logger: Logger, args: any, cb: (err?: any) => void): void {
```

Comment:
Instead of `args: any`, we should use `args: CommandArgs`, which would allow us to catch the issue on line 33